qa/suites/rgw/archive-zone/tasks/test_archive.yaml

```diff
@@ -0,0 +1,26 @@
+# see http://tracker.ceph.com/issues/20360 and http://tracker.ceph.com/issues/18126
+os_type: centos
+
+tasks:
+- install:
+- ceph: {cluster: c1}
+- ceph: {cluster: c2}
+- rgw:
+    c1.client.0:
+      port: 8000
+      valgrind: [--tool=memcheck, --max-threads=1024] # http://tracker.ceph.com/issues/25214
+    c1.client.1:
+      port: 8001
+      valgrind: [--tool=memcheck, --max-threads=1024]
+    c2.client.0:
+      port: 8000
+      valgrind: [--tool=memcheck, --max-threads=1024]
+    c2.client.1:
+      port: 8001
+      valgrind: [--tool=memcheck, --max-threads=1024]
+- rgw-multisite:
+- rgw-multisite-tests:
+    args:
+    - tasks.rgw_multi.tests_az
```

Comment:
other than the new realm description in `realms/two-zone-archive.yaml`, this `tasks.rgw_multi.tests_az` line seems to be the only difference from the rgw/multisite suite. do you think there's a way to put `two-zone-archive.yaml` under `rgw/multisite/realms/` and teach the rgw_multisite_tests.py task (or the rgw_multi tests themselves) how to filter the tests based on cluster configuration?

i'd like to make a similar change for pubsub tests, and would rather not duplicate the entire suite again

Reply:
@cbodley  I have removed the archive yamls from here. Will open a new PR for test fixes. I think it is safe to merge.